dist/messages/messages.json

```diff
@@ -4,8 +4,8 @@
     "wrk_dir_label": "Working directory: ",
     "cmd_label": "Command line: ",
     "cmd_cannot_be_empty": "Command line cannot be empty.",
-    "failed_run_sa": "Run has been failed due to static analysis violations reported",
-    "failed_run_ut": "Run has been failed due to test failures reported",
-    "failed_run_non_zero": "Run has been failed due to non-zero dotTEST exit code: ",
+    "failed_run_sa": "dotTEST run failed because static analysis violations were reported",
+    "failed_run_ut": "dotTEST run failed because failed unit tests were reported",
+    "failed_run_non_zero": "dotTEST run failed with a non-zero dotTEST exit code:",
```

Comment:
Please keep white space characters. This line is missing space after colon

Reply:
Fixed.

Reply:
Thanks!